packages/z/zed/package.yml

```diff
@@ -0,0 +1,42 @@
+name       : zed
+version    : 0.139.1
+release    : 1
+source     :
+    - git|https://github.com/zed-industries/zed.git : v0.139.1-pre
+homepage   : https://zed.dev/
+license    :
+    - AGPL-3.0-or-later
+    - Apache-2.0
+    - GPL-3.0-or-later
+component  : editor
+summary    : Code at the speed of thought
+description: |
+    Zed is a high-performance, multiplayer code editor from the creators of Atom and Tree-sitter. It's also open source.
+networking : yes
+builddeps  :
+    - pkgconfig(alsa)
+    - pkgconfig(fontconfig)
+    - pkgconfig(xkbcommon)
+    - mold
+    - rust
+setup      : |
+    APP_NAME="Zed" DO_STARTUP_NOTIFY="true" APP_ICON="zed" envsubst < crates/zed/resources/zed.desktop.in > zed.desktop
+
+    # Doesn't link with LLD 18
```

Comment:
Please add a comment with what the specific build error is so that someone can check if it still occurs in the future.